lib/docker.ts

```diff
@@ -47,6 +48,14 @@ class Docker {
       password,
       loadImage: false,
       imageSavePath,
+      reqOptions: {
```

Comment:
So if I understand correctly `snyk-docker-pull` will use the `reqOptions` to understand if the image is OCI and pull accordingly?

Reply:
`snyk-docker-pull` will pass these accept headers all the way through to the actual request to the OCI distribution spec compliant registry, which will therefore sometimes respond with an OCI image. Additional changes were made in `snyk-docker-pull` to ensure it does correctly handle these images.